packages/esm-home-app/src/index.ts

```diff
@@ -24,6 +24,8 @@ export const homeWidgetDbLink = getSyncLifecycle(createDashboardLink(dashboardMe
 
 export const homeWidgetDashboard = getSyncLifecycle(homeWidgetDashboardComponent, options);
 
+export const metricsSlot = getAsyncLifecycle(() => import('./metrics/metrics.component'), options);
```

Comment:
This can be named better @denniskigen should we just call this `metrics`,`metricsContainer` or `MetricsComponent`

```suggestion
export const metricsContainer = getSyncLifecycle(MetricsComponent, options);
```

Reply:
This is resolved.